nikola/filters.py

```diff
@@ -268,8 +268,11 @@ def minify_lines(data):
 
 def _run_typogrify(data, typogrify_filters, ignore_tags=None):
     """Run typogrify with ignore support."""
+    default_ignore_tags = ["title", ".math"]
     if ignore_tags is None:
-        ignore_tags = ["title", ".math"]
+        ignore_tags = default_ignore_tags
+    else:
+        ignore_tags.extend(default_ignore_tags)
```

Comment:
I’d prefer not to mutate this list and instead use:

    ignore_tags = ignore_tags + default_ignore_tags